homeassistant/components/lcn/__init__.py

```diff
@@ -126,16 +136,18 @@ async def async_unload_entry(hass, config_entry):
 class LcnEntity(Entity):
     """Parent class for all entities associated with the LCN component."""
 
-    def __init__(self, config, entry_id, device_connection):
+    def __init__(
+        self, config: ConfigType, entry_id: str, device_connection: DeviceConnectionType
+    ) -> None:
         """Initialize the LCN device."""
         self.config = config
         self.entry_id = entry_id
         self.device_connection = device_connection
         self._unregister_for_inputs = None
-        self._name = config[CONF_NAME]
+        self._name = str(config[CONF_NAME])
```

Comment:
Add a a type hint instead:
```suggestion
        self._name: str = config[CONF_NAME]
```